tests/core/Jalasoft.TeamUp.Projects.Core.Tests/InvitationsServiceTests.cs

```diff
@@ -108,5 +109,23 @@ public void GetInvitationsByProjectId_InvitationsDontExist_EmptyArray()
             // Assert
             Assert.Empty(result);
         }
+
+        [Fact]
+        public void PostInvitation_Return_Invitation()
```

Comment:
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices please follow the naming convention adopted by the team in the other unit tests